typescript/common-resolvers/src/iog-inference.ts

```diff
@@ -79,52 +130,114 @@ const runIog = async (
   //   ],
   // };
   // End dummy iog
-  if (result) {
-    const geometry = {
-      type: "Polygon",
-      coordinates: result?.polygons,
-    };
-    const { smartToolInput } = await throwIfResolvesToNil(
-      "No label with such id",
-      repository.label.get
-    )({ id: args.data.id }, user);
+  const geometry = {
+    type: "Polygon",
+    coordinates: result?.polygons as number[][][],
+  };
 
-    const xCoordinates = geometry.coordinates.reduce(
-      (xCoordinatesCurrent: number[], polygon: number[][]) => [
-        ...xCoordinatesCurrent,
-        ...polygon.map((point: number[]) => point[0]),
-      ],
-      []
-    );
-    const yCoordinates = geometry.coordinates.reduce(
-      (yCoordinatesCurrent: number[], polygon: number[][]) => [
-        ...yCoordinatesCurrent,
-        ...polygon.map((point: number[]) => point[1]),
-      ],
-      []
-    );
-    const [x, y, X, Y] = [
-      Math.min(...xCoordinates),
-      Math.min(...yCoordinates),
-      Math.max(...xCoordinates),
-      Math.max(...yCoordinates),
-    ];
-    const width = X - x;
-    const height = Y - y;
+  const xCoordinates = geometry.coordinates.reduce(
+    (xCoordinatesCurrent: number[], polygon: number[][]) => [
+      ...xCoordinatesCurrent,
+      ...polygon.map((point: number[]) => point[0]),
+    ],
+    []
+  );
+  const yCoordinates = geometry.coordinates.reduce(
+    (yCoordinatesCurrent: number[], polygon: number[][]) => [
+      ...yCoordinatesCurrent,
+      ...polygon.map((point: number[]) => point[1]),
+    ],
+    []
+  );
+  const [x, y, X, Y] = [
+    Math.min(...xCoordinates),
+    Math.min(...yCoordinates),
+    Math.max(...xCoordinates),
+    Math.max(...yCoordinates),
+  ];
+  const width = X - x;
+  const height = Y - y;
```

Comment:
Nitpicking:
I feel like this could be further optimised. We could do this in a single loop/reduce.

But feel free to ignore this comment if you don't feel like it would change much.

Reply:
Done!